.travis.yml

```diff
@@ -7,11 +7,9 @@ cache:
     - $HOME/.composer/cache/files
 
 php:
-  - 5.3
```

Comment:
Changed the composer.json now. Thanks, let me know if it is not the correct way.